src/ImageSharp/Color/Color.cs

```diff
@@ -27,19 +27,19 @@ namespace SixLabors.ImageSharp
         private Color(byte r, byte g, byte b, byte a)
         {
             this.data = new Rgba64(
-                ImageMaths.UpscaleFrom8BitTo16Bit(r),
-                ImageMaths.UpscaleFrom8BitTo16Bit(g),
-                ImageMaths.UpscaleFrom8BitTo16Bit(b),
-                ImageMaths.UpscaleFrom8BitTo16Bit(a));
+                ImageMath.UpscaleFrom8BitTo16Bit(r),
```

Comment:
I would move this to `Numerics` too. Or to a separate `ColorNumerics` (or similarly named) class.